src/Interfaces/AbstractPolyhedron_functions.jl

```diff
@@ -879,3 +879,42 @@ function an_element(P::AbstractPolyhedron{N}) where {N<:Real}
     e₁ = SingleEntryVector(1, n, one(N))
     return σ(e₁, P)
 end
+
+"""
+    isbounded(P::AbstractPolyhedron{N}; solver=LazySets.default_lp_solver(N)) where {N}
+
+Determine whether a polyhedron is bounded.
+
+### Input
+
+- `P`       -- polyhedron
+- `backend` -- (optional, default: `default_lp_solver(N)`) the backend used
+               to solve the linear program
+
+### Output
+
+`true` iff the polyhedron is bounded
+
+### Algorithm
+
+The algorithm is based on Stiemke's theorem of alternatives.
+
+Proposition 1. If ``ker(A)≠\\{0\\}``, then ``P`` is unbounded.
+
+Proposition 2. Assume that ``ker(A)={0}`` and ``P`` is non-empty.
```

Comment:
In the code you do not check for emptiness. Does the code also work if `P` is empty? Maybe add such a test.

Reply:
the code doesn't check if the input is an empty set. i checked and the answer is the same as for `isbounded_unit_dimensions` (= false).

Reply:
i will add this assumption to the docs nevertheless

Reply:
I hope you mean `true`!
```julia
julia> isbounded_unit_dimensions(HPolyhedron([HalfSpace([1.], 0.), HalfSpace([-1.], 1.)]))
true
```

Reply:
yes. though for some cases it gives an error and Stiemke gives "false"

```julia
p = HPolyhedron([HalfSpace(x <= 1, var), HalfSpace(x >= 2, var)])

__isbounded_unit_dimensions(p)
the support vector is undefined because the polyhedron is empty

__isbounded_stiemke(p)
false
```
it is tempting to make the emptiness check in the function, but the cost is relatively high,

```julia
p = convert(HPolyhedron, rand(Hyperrectangle, dim=10));
@btime isempty($p)
@btime __isbounded_stiemke($p)

  22.653 μs (79 allocations: 7.45 KiB)
  30.743 μs (100 allocations: 20.56 KiB)
true
```

Reply:
we can also let the check be optional, defaulting to `true`. what are your thoughts?

Reply:
Yes, getting a wrong answer by default doesn't sound good.

Reply:
i had changed the docstring to state that the condition has to be checked by the user.